### ISSUE TITLE
Duplicate project

### ISSUE THREAD

**Issue by raintonr** [googlecodelabs/smarthome-local#2]
Forgive me, but isn't this whole project a duplicate of...

https://github.com/actions-on-google/smart-home-local

... not that I can get either of these to work :(

**Comment by devunwired** [comment]
This repository contains the companion project for the [local execution codelab](https://codelabs.developers.google.com/codelabs/smarthome-local).
The repository you linked contains the Getting Started sample code.

If you believe you've found an issue with the code in this repository, feel free to file an issue. For general development questions with Google Smart Home or Local Home SDK, please post on our community forums:
Reddit: https://www.reddit.com/r/GoogleAssistantDev/
StackOverflow: https://stackoverflow.com/questions/tagged/google-smart-home

**Comment by raintonr** [comment]
Thanks. I have commented there (https://github.com/actions-on-google/smart-home-local/issues/16#issuecomment-576155117)

FWIW, the issue I was having with this project was to do with device linking and think that was not working due to a bug in the Google Home Android app. I tried many times using my phone but it just wouldn't link. On my phone, a page was opening and loading the dummy OAuth page but then nothing. However, just on a whim I tried on another device (an old Android tablet I have) and it worked first time there, loaded the dummy OAuth page then redirected correctly and called the SYNC service.

But why did this work on the tablet?

I was having a few issues with the Actions Console and GCP part of the project. I had mistakenly put the wrong  region in the Oauth/Fullfilmane URLs in the Actions Console. When I tried linking from my phone realised this when I couldn't see anything happening in the GCP log. I then updated the URLs in the Actions Console but it still wouldn't fully work on my phone (I could then see the Oauth redirect but no Sync).

I think somehow, the Google Home Android app on my phone (or underlying browser stack that is used to load the OAuth page) had somehow cached something with the wrong URLs and therefore it wouldn't work.

I should also mention that on my phone, the test app showed twice in the Google Home Android app services list but neither of them would work.

I am not completely sure this is the cause, but either way, the logging/error handing in the Google Home Android app isn't great and if it was certainly something with that app in my phone that stopped this example working for me last night. However, I don't know where to report those issues, is there a place?
